src/main.c

```diff
@@ -699,9 +701,21 @@ static const char *listener_setup_ssl_picotls(struct listener_config_t *listener
         key = SSL_get_privatekey(fakeconn);
         assert(key != NULL);
         cert = SSL_get_certificate(fakeconn);
+        /* obtain peer verify mode */
+        verify_mode = (SSL_get_verify_mode(fakeconn) & SSL_VERIFY_PEER)? 1 : 0;
         SSL_free(fakeconn);
     }
 
+    if (verify_mode == 1) {
```

Comment:
```suggestion
    if (verify_mode) {
```
It's our convention to not do `== 1` or `== 0` against boolean variables.